django/econsensus/publicweb/templates/decision_list.html

```diff
@@ -66,8 +66,9 @@
                             {% if tab != 'actionitems' %}
                                 <td class="id"><a href="{% url 'publicweb_item_detail' object.id %}"><span class="iconified">{{ object.id }}</span></a></td>
                                 <td>
-                                    <a href="{% if request.user|is_watching:object %}{% url 'remove_watcher' object.id %}{% else %}{% url 'add_watcher' object.id %}{% endif %}?next={{ request.path_info }}">
-                                        <input type="checkbox"{% if request.user|is_watching:object %} checked="True"{% endif %} />
+                                    <a class='toggle watch'
+                                       href="{% if request.user|is_watching:object %}{% url 'remove_watcher' object.id %}{% else %}{% url 'add_watcher' object.id %}{% endif %}?next={{ request.path_info }}">
+                                        {% if request.user|is_watching:object %}&#x2611;{% else %}&#x2610;{% endif %}
```

Comment:
An alternative approach would be to replace the utf8 tickbox with an image with an 'alt' attribute, as this would be read by screen readers. Cleaner than the option above, though it uses more bandwidth.